modules/administration-guide/partials/assembly_enabling-readiness-init-containers.adoc

```diff
@@ -5,11 +5,9 @@
 
 :context: assembly_enabling-startup-init-containers
 
-Normally, {prod-short} Operator installs {prod} and starts its containers in the right order.
-However, if a node with {prod-short} gets restarted, than all {prod-short} containers start simultaneously and some might fail as dependent component might not be ready yet.
-To handle such a case, one should enable readiness init containers for {prod-short} components.
+{prod-short} Operator installs {prod-short} and starts its containers in the right order. However, suppose a node with {prod-short} gets restarted. In that case, all {prod-short} containers start simultaneously, causing some containers to fail because of the dependent component that might not be ready yet. To handle such a case,  enable readiness init containers for {prod-short} components.
 
-Choose the procedure depending on installer type:
+To troubleshoot this situation, enable readiness init containers by following the procedure that matches the used installation method:
```

Comment:
I am not sure that `troubleshoot` is the right work in this case. It is more an option to enable if needed.